src/store/modules/game.js

```diff
@@ -66,7 +66,6 @@ export default {
 			if (Object.hasOwnProperty.call(newGame, 'p1Ready')) state.p1Ready = newGame.p1Ready;
 			if (Object.hasOwnProperty.call(newGame, 'passes')) state.passes = newGame.passes;
 			if (Object.hasOwnProperty.call(newGame, 'players')) state.players = _.cloneDeep(newGame.players);
-			if (Object.hasOwnProperty.call(newGame, 'srap')) state.scrap = _.cloneDeep(newGame.scrap);
```

Comment:
Nice -- looks like you caught a typo (`srap`). Did you mean to replace this instead of removing it?

Reply:
There is already a line of code for scrap several lines above. Sorry, should've mentioned this earlier :)